ansible/group_vars/all

```diff
@@ -90,7 +90,7 @@ controller:
   managedFraction: "{{ controller_managed_fraction | default(1.0 - (controller_blackbox_fraction | default(__controller_blackbox_fraction))) }}"
   blackboxFraction: "{{ controller_blackbox_fraction | default(__controller_blackbox_fraction) }}"
   timeoutFactor: "{{ controller_timeout_factor | default(2) }}"
-  timeoutAddon: "{{ controller_timeout_addon | default(1) }}"
+  timeoutAddon: "{{ controller_timeout_addon | default('1 m') }}"
```

Comment:
You also need to feed the new setting into the controller deployment like so:

https://github.com/apache/openwhisk/blob/78b2cbd7bd00fb6313ead1d71a316256def8b0c4/ansible/roles/controller/tasks/deploy.yml#L254-L255

@dgrove-oss what's the usual approach to add such configuration parameters to other deployment approaches of Openwhisk - like Kubernetes?

Reply:
@sven-lange-last - Kubernetes uses the same mechanism of setting an environment variable like `CONFIG_whisk_loadbalancer_timeoutFactor` to override the default value that comes from loading `reference.conf`.  To make it easy to configure, you would add an entry into values.yaml (for example like this:https://github.com/apache/openwhisk-deploy-kube/blob/master/helm/openwhisk/values.yaml#L99) and use that entry in the `env` section of controller-pod.yaml (for example like this: https://github.com/apache/openwhisk-deploy-kube/blob/master/helm/openwhisk/templates/controller-pod.yaml#L132)